Sources/Hyperspace/Service/Transport/TransportService.swift

```diff
@@ -42,19 +43,25 @@ extension TransportService: Transporting {
             self?.networkActivityController?.stop()
             self?.handle(data: data, response: response, error: error, for: HTTP.Request(urlRequest: request), completion: completion)
         }
-        
-        tasks[request] = task
-        task.resume()
-        
+
+        queue.async { [weak self] in
```

Comment:
I have less strong feelings on this one to be honest, but is this intentionally `async` ?

Reply:
Works for me